src/components/TasksTable/index.jsx

```diff
@@ -54,10 +74,19 @@ const assignments = Object.values(ASSIGNEE);
   filter: {
     ...theme.mixins.gutters(),
   },
+  infoButton: {
+    padding: '0px 4px',
+    minWidth: 'auto',
+  },
+  drawer: {
+    width: 250,
+  },
```

Comment:
I have the impression the description is a little bit squeezed. I would like to see a bigger drawer, perhaps 400px? Let's also include a "X" icon in the top right direction of the drawer. By doing that, we can have the drawer take full width for mobile and let the user click on "X" to close the drawer.